examples/compose/compose.yml

```
@@ -2,10 +2,10 @@ name: addy
 
 services:
   db:
-    image: mariadb:10.5
+    image: mariadb:11.3
```

I think we can use major

```suggestion
    image: mariadb:11
```